MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/offline/CacheManagementActivity.java

```diff
@@ -0,0 +1,284 @@
+    /*firebaseRemoteConfig.setConfigSettingsAsync(new FirebaseRemoteConfigSettings.Builder()
+        .setMinimumFetchIntervalInSeconds(5)
+        .build());*/
+
+    // Set default Remote Config parameter values. The app uses the in-app default values, and
+    // when you need to adjust those defaults, you set an updated value for only the values you
+    // want to change in the Firebase console.
+    firebaseRemoteConfig.setDefaults(R.xml.cache_management_remote_config_defaults);
+
+    // Fetch the latest Remote Config values from Firebase and then adjust the cache if needed.
+    firebaseRemoteConfig.fetchAndActivate().addOnCompleteListener(this,
+        new OnCompleteListener<Boolean>() {
+          @Override
+          public void onComplete(@NonNull Task<Boolean> task) {
+            if (task.isSuccessful()) {
+              adjustCacheBasedOnRemoteConfigServerSettings();
+            } else {
+              Logger.d("CacheManagementActivity", "Couldn\'t get Firebase cache "
+                  + "management remote config settings from Firebase.");
+            }
+          }
+        });
+
+    mapView = findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(new OnMapReadyCallback() {
+      @Override
+      public void onMapReady(@NonNull MapboxMap mapboxMap) {
+        mapboxMap.setStyle(Style.MAPBOX_STREETS, new Style.OnStyleLoaded() {
+          @Override
+          public void onStyleLoaded(@NonNull Style style) {
+
+            findViewById(R.id.invalidate_ambient_cache_button).setOnClickListener(new View.OnClickListener() {
+              @Override
+              public void onClick(View view) {
+                invalidateAmbientCache();
+              }
+            });
+
+            findViewById(R.id.set_max_ambient_cache_size_button).setOnClickListener(new View.OnClickListener() {
+              @Override
+              public void onClick(View view) {
+                EditText newMaxAmbientCacheSizeTextView = findViewById(
+                    R.id.set_max_ambient_cache_size_button_editText);
+                long newMaxAmbientCacheSize = Long.valueOf(
+                    newMaxAmbientCacheSizeTextView.getText().toString());
+                setNewMaxAmbientCache(newMaxAmbientCacheSize);
+              }
+            });
+
+            findViewById(R.id.clear_ambient_cache_button).setOnClickListener(new View.OnClickListener() {
+              @Override
+              public void onClick(View view) {
+                clearAmbientCache();
+              }
+            });
+
+            findViewById(R.id.reset_database_button).setOnClickListener(new View.OnClickListener() {
+              @Override
+              public void onClick(View view) {
+                resetDatabase();
+              }
+            });
+          }
+        });
+      }
+    });
+  }
+
+  /**
+   * Invalidate the ambient cache. See ANDROID-DOCS-URL-HERE for more
```

Comment:
Missing the doc? :)
Same for below occurrences.

Reply:
Fixed. All changed to https://docs.mapbox.com/android/maps/overview/offline/#cache-management, which will be the correct URL when https://github.com/mapbox/android-docs/pull/995 lands.